app/components/Intercept_Components/InterceptTextBox.tsx

```diff
@@ -2,12 +2,19 @@ import * as React from "react";
 import {RequestHeaderList} from "./RequestHeaderList";
 
 export const InterceptTextBox = props => {
+  const isObject = (val:any) => {
```

Comment:
A simpler check for this would be:

```javascript
return !value && typeof val === 'object';
```

So that would return true only if the value is truthy, and it's a type of object. `null` is also an object, but that's covered in the first check. stringify of a function is pretty much a noop, so that check is not required, we can treat it as not an object.

<img width="511" alt="screenshot 2018-05-04 14 08 39" src="https://user-images.githubusercontent.com/400299/39619221-aa8a3fb2-4fa4-11e8-841f-8e132516054d.png">